service/rtc/sfu.go

```diff
@@ -254,9 +259,17 @@ func (s *Server) InitSession(cfg SessionConfig, closeCb func() error) error {
 			})
 
 			for {
-				rtp, _, readErr := remoteTrack.ReadRTP()
-				if readErr != nil {
-					s.log.Error("failed to read RTP packet", mlog.Err(readErr))
+				buf := s.bufPool.Get().([]byte)
+				i, _, err := remoteTrack.Read(buf)
+				if err != nil {
+					s.log.Error("failed to read RTP packet", mlog.Err(err))
+					s.metrics.IncRTCErrors(us.cfg.GroupID, "rtp")
+					return
+				}
+
+				rtp := &rtp.Packet{}
```

Comment:
wait, I need to check if this escapes to heap. And if there's a way to make it non-escaping. But that shouldn't block the load tests.

Reply:
Does not escape. All good. :+1: